doc/dasd.md

```diff
@@ -0,0 +1,75 @@
+
+DASDs
+-----
+
+There are ECKD and FBA DASDs.
+
+
+ECKD DASDs
+----------
+
+Can be CDL or LDL formatted.
+
+
+CDL formatted ECKD DASDs
+------------------------
+
+Most common type.
+
+Can be used with a DASD partition table.
+
+Cannot be used as a block device since the two first tracks are special.
+
+- Partition Types: DASD
+
+- Use as Block Device: No
+
+
+LDL formatted ECKD DASDs
+------------------------
+
+Implicit partition table with one big partition.
+
+Can be used directly as a block device. But since the implicit partition stays
+this looks like a recipe for disaster.
+
+- Partition Types: Implicit
+
+- Use as Block Device: Unsupported
+
+
+Unformatted ECKD DASDs
+----------------------
+
+- Partition Types: None
+
+- Use as Block Device: No
+
+
+FBA DASDs
+---------
+
+Can be used with a MSDOS or GPT partition table. Otherwise implicit partition
```

Comment:
Question: I'm not a storage expert, but this looks a bit confusing to me. It can be used with MSDOS or GTP PT but cannot be used  as a block device? I thought an MSDOS partition *is* a block device... So how you can use it actually?

Reply:
The partitions on the MSDOS, GPT or imlicit partition table can be used as block devices. But the DASD itself cannot.